model-optimizer/extensions/front/tf/argmin_ext.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.ops.arg_ops import ArgMaxOp
+from mo.front.extractor import FrontExtractorOp
+from mo.front.tf.extractors.utils import tf_dtype_extractor
+
+
+class ArgMinFrontExtractor(FrontExtractorOp):
+    op = 'ArgMin'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        attrs = {
+            'top_k': 1,
+            'axis': None,
+            'keepdims': 0,
+            'remove_values_output': True,
+            'output_type': tf_dtype_extractor(node.pb.attr['out_type'].type, np.int64)
```

Comment:
The TF operation attribute name is `output_type` not `out_type`. The same comment for the `ArgMax` operation